src/helpers.ts

```diff
@@ -113,6 +122,20 @@ export function interpretIdentifier(identifier: string): { address: string; publ
   }
 }
 
+export async function signMetaTxData(
+  identity: string,
+  signerAddress: string,
+  privateKeyBytes: Uint8Array,
```

Comment:
It would be nice to be able to use a signer function instead of relying on a private key here, but I'm not sure ethers provides this abstraction.

Reply:
I see what you mean. Maybe instead of passing the `privateKeyBytes` in there we should replace it with an instantiated `SigningKey` from ethersproject? This method would then just call `signDigest()` on the passed param.